spec/controllers/coronavirus_form/data_export_controller_spec.rb

```diff
@@ -0,0 +1,65 @@
+# frozen_string_literal: true
+
+require "spec_helper"
+
+RSpec.describe CoronavirusForm::DataExportController, type: :controller do
+  subject(:instance) { described_class.new }
+
+  describe "#usage_statistics" do
+    let(:start_date) { "2020-04-10" }
+    let(:end_date) { "2020-04-15" }
+
+    before do
+      FormResponse.create(form_response: { able_to_leave: "Yes", get_food: "Yes" }, created_at: "2020-04-10 10:00:00")
+      FormResponse.create(form_response: { able_to_leave: "Yes", get_food: "No" }, created_at: "2020-04-10 10:00:00")
+      FormResponse.create(form_response: { able_to_leave: "Yes", get_food: "Yes" }, created_at: "2020-04-11 10:00:00")
+      FormResponse.create(form_response: { able_to_leave: "No", get_food: "No" }, created_at: "2020-04-11 10:00:00")
+    end
+
+    it "returns a JSON representation of the form responses" do
```

Comment:
Technically it returns a ruby hash, rather than a JSON representation.

Reply:
Ah, yes.  Updated the test description.